master/internal/api_project_intg_test.go

```diff
@@ -784,3 +784,104 @@ func TestGetProjectByID(t *testing.T) {
 	require.Equal(t, projectName, project.Name)
 	require.Equal(t, resp.Project.Id, project.Id)
 }
+
+func TestGetMetadataValues(t *testing.T) {
+	api, curUser, ctx := setupAPITest(t, nil)
+	_, projectIDInt := createProjectAndWorkspace(ctx, t, api)
+	projectID := int32(projectIDInt)
+	exp := createTestExpWithProjectID(t, api, curUser, int(projectID))
+
+	task := &model.Task{TaskType: model.TaskTypeTrial, TaskID: model.NewTaskID()}
```

Comment:
nit (non-blocking): Would it be worth turning this into a loop or a helper function?

Reply:
Done